ionoscloud/resource_server_cube_test.go

```diff
@@ -706,3 +734,64 @@ resource ` + constant.ServerCubeResource + ` ` + constant.ServerTestResource + `
     }
 }
 ` + ServerImagePassword
+
+const testAccCheckCubeServerConfigPrimaryBootVolume = `
```

Comment:
@cbeti-ionos this configuration is not used anywhere

Reply:
placeholder, should be updated now